draft-demarco-oauth-status-assertions.md

```diff
@@ -235,6 +235,9 @@ such continuous surveillance.
 The general requirements for the implementation of Status Assertion are
 listed in this section. The Status Assertion:
 
+- notifies the Holder about the status of their Digital Credential,
+because the Holder MUST be informed by the Issuers of any changes
```

Comment:
```suggestion
- notifies the Holder about the status of their Digital Credential. Issuers MUST notify the Holder about the status of their Digital Credential because the Holder needs to be informed of any changes.
```

Reply:
@marinaado wdyt? ^

Reply:
I think it would be slightly nicer if it says: "The Issuer MUST notify" instead "Issuers MUST notify." 

I searched through the specification and i found the we use singular with definite article for BCP 14 words. 

Otherwise is good.

Reply:
done, thank you